sdks/java/core/pom.xml

```diff
@@ -315,6 +315,19 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- 
+    bytebuddy has provided scope dep on SuppressFBWarnings 
+    annotations (to avoid a real dep) and these are
+    retained on certain .class files, necessitating
+    this provided dependency to instantiate those classes
+    -->
+    <dependency>
```

Comment:
@davorbonaci can you clarify Apache licensing here? The root cause of our needing something here is a similar construct in the bytebuddy pom: https://github.com/raphw/byte-buddy/blob/master/pom.xml#L163